src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -93,5 +105,35 @@ private string GetProjectJsonPath(string projectJson)
 
             throw new Exception($"Unable to find project file at {projectJson}");
         }
+
+        private IEnumerable<string> GetProjectsFromGlobalJson(string globalJson)
+        {
+            if (!File.Exists(globalJson))
+            {
+                throw new Exception($"Unable to find global settings file at {globalJson}");
+            }
+
+            var searchPaths = ProjectDependencyFinder.GetGlobalPaths(Path.GetDirectoryName(globalJson));
```

Comment:
If this is the reason for ProjectDependencyFinder to be public, maybe we should move this one API to its own class and only have that one public. Or better yet, does NuGet have an API for this?